### PR TITLE
Add XING and kununu

### PR DESCRIPTION
**Pull request by jmagalhaes-xing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Tech Companies in Portugal :portugal:
 
-This is a opinionated list of some technology companies operating in Portugal.  
+This is a opinionated list of some technology companies operating in Portugal.
```
Comment (marmelo) [inline README.md]: Please do not remove the trailing whitespace.  
They will create a line break.

File: README.md [hunk 2]
```diff
@@ -66,6 +66,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
 | [Critical Manufacturing](https://www.criticalmanufacturing.com) [:rocket:](https://www.criticalmanufacturing.com/en/careers) | Manufacturing business-critical applications. | `Porto` |
+| [kununu](https://www.kununu.com/) [:rocket:](https://corporate.xing.com/en/career/porto/) | Workplace insights that matter. | `Porto` |
```
Comment (marmelo) [inline README.md]: Is this careers link shared with XING?

Comment (jmagalhaes-xing) [reply README.md]: Yes, both XING and kununu brands belong to XING SE.

**Comment by pedrommenezes** [comment]
👍

**Comment by vpereira01** [comment]
ping @jmagalhaes-xing

File: README.md (hunk 1 above)
Comment (vpereira01) [reply README.md]: @jmagalhaes-xing I think you might have forgot this comment.

**Comment by vpereira01** [comment]
This PR breaks the build because of the duplicated link to the careers pages, not sure what's the best fix.

File: README.md (hunk 1 above)
Comment (jmagalhaes-xing) [reply README.md]: Fixed again. I might have removed it again on the merges

**Comment by marmelo** [comment]
Please add `--allow-dupe` argument to `awesome_bot` inside the `.travis.yml` file.  
Something like:
```
awesome_bot -f README.md --allow-ssl --allow-redirect --allow-dupe -w www.linkedin.com,www.hostelworldgroup.com,www.olxgroup.com
```

**Comment by marmelo** [comment]
Or else I can do this in a later commit.

File: README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.com/marmelo/tech-companies-in-portugal.svg?branch=master)](https://travis-ci.com/marmelo/tech-companies-in-portugal)
 [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/marmelo/tech-companies-in-portugal/blob/master/LICENSE)
 
-This is a opinionated list of some technology companies operating in Portugal.  
+This is a opinionated list of some technology companies operating in Portugal.
```
Comment (marmelo) [inline README.md]: Two white spaces, please.

**Comment by marmelo** [comment]
Thanks @jmagalhaes-xing @vpereira01!  
I will fix the duplicated links next.

**Pull request merged**